bin/anthology/volumes.py

```diff
@@ -40,9 +41,10 @@ def __init__(self,
         """
         self.collection_id = collection_id
         self._id = volume_id
+        self.ingest_date = ingest_date
         self.formatter = formatter
         self._set_meta_info(meta_data)
-        self.attrib["venues"] = venue_index.register(self)
+        self.attrib["venues"] = venue_index.get_associated_venues(self.full_id)
         self.attrib["sigs"] = sig_index.get_associated_sigs(self.full_id)
```

Comment:
Currently, creating a volume causes a side effect of registering that volume. I factored this out so there is no more side effect.